apps/ws/src/GameManager.ts

```diff
@@ -85,6 +90,12 @@ export class GameManager {
         const game = this.games.find((game) => game.gameId === gameId);
         if (game) {
           game.makeMove(user, message.payload.move);
+          game.clearTimer();
+          const timer = setTimeout(() => {
+            game.endGame();
+            this.removeGame(game.gameId);
+          }, 60 * 1000);
```

Comment:
note for later, move this some place else this smells a bit
make time configurable